### PR TITLE
Formatting cleanups, fixed all warnings in markdowner.py and adapted GH Action to no longer allow warnings (3.21)

### PR DESCRIPTION
**Pull request by olehermanse**

### CODE REVIEW

**Review by craigcomstock — APPROVED** [review]

File: resources/additional-topics/hierarchies.markdown [hunk 1]
```diff
@@ -33,7 +33,8 @@ on demand to other nodes, without any particular ranking. If you move in a mesh,
 you cannot easily measure how far you are away from a given point, as their
 might be more than one way of getting there.
 
-Mesh architectures are often robust to failure as there can be multiple _peer to peer_ routes for passing messages or information.
+Mesh architectures are often robust to failure as there can be multiple
+_peer to peer_ routes for passing messages or information.
```
Comment (craigcomstock) [inline resources/additional-topics/hierarchies.markdown:37]: why the column breaking a sentence? don't we prefer a sentence-per-line?

Comment (craigcomstock) [reply resources/additional-topics/hierarchies.markdown:37]: ah, I see this file's preceding content is column-ized so 🤷 👍

**Review by craigcomstock — APPROVED** [review]

**Comment by olehermanse** [comment]
@cf-bottom jenkins, please

**Comment by cf-bottom** [comment]
Alright, I triggered a build:

[Build Status](https://ci.cfengine.com//job/fast-build-and-deploy-docs-3.21/87/)

**Jenkins:** https://ci.cfengine.com/job/fast-build-and-deploy-docs-3.21/87/

**Documentation:** http://buildcache.cfengine.com/packages/build-documentation-pr/jenkins-fast-build-and-deploy-docs-3.21-87/output/_site/

**Pull request merged**